File: C++/src/Solution501_600/Solution561.cpp
//
// Created by windows on 2023/3/2.
//

#include "Solution561.h"
#include <algorithm>

int Solution561::arrayPairSum(vector<int> &nums) {
    sort(nums.begin(), nums.end());
    int sum = 0;
    for (int i = 0; i < nums.size(); i += 2) {
        sum += nums[i];
    }
    return sum;
}

File: Java/src/main/java/QuestionBank/Solution501_600/Solution561.java
package QuestionBank.Solution501_600;

import java.util.Arrays;

/**
 * @author windows
 */
public class Solution561 {
    public int arrayPairSum(int[] nums) {
        Arrays.sort(nums);
        int sum = 0;
        for (int i = 0; i < nums.length; i+=2) {
            sum += nums[i];
        }
        return sum;
    }


    public static void main(String[] args) {
        Solution561 solution561 = new Solution561();
    }
}